Client/Frontend/InternalSchemeHandler/ErrorPageHelper.swift

```diff
@@ -310,7 +318,8 @@ extension ErrorPageHelper: TabContentScript {
                 let host = originalURL.host {
                 let origin = "\(host):\(originalURL.port ?? 443)"
                 certStore?.addCertificate(cert, forOrigin: origin)
-                _ = message.webView?.reload()
+            message.webView?.evaluateJavaScript("location.replace('\(originalURL.absoluteString)')", completionHandler: nil)
```

Comment:
Again, make sure the string is safe.

Reply:
I commonized all this code into a webview.replaceLocation() that makes all the usages safe